src/components/navigation/navigation.svelte

```diff
@@ -16,11 +14,44 @@
   {#if $$slots.actions}
     <slot name="actions" />
   {/if}
+
+  <div class="active-indicator" />
 </div>
 
 <style lang="scss">
+  :host, .leo-navigation {
+    --anchor-padding: var(--leo-spacing-m);
+    position: relative;
+  }
+
+  :global(leo-navigation::before), .active-indicator {
+    // Anchor positioned active indicator - we have a fallback in navigationItem
+    // for when this isn't supported.
+    @supports (anchor-name: --active-indicator) {
+      position-anchor: --active-indicator;
+
+      transition:
+        top 0.12s ease-in-out,
+        bottom 0.12s ease-in-out,
+        left 0.12s ease-in;
+
+      content: '';
+      width: 4px;
+      border-top-right-radius: var(--leo-radius-xs);
+      border-bottom-right-radius: var(--leo-radius-xs);
+      background: var(--leo-color-text-interactive);
+      position: absolute;
+      left: anchor(left);
+      top: calc(anchor(top) + var(--anchor-padding));
+      bottom: calc(anchor(bottom) + var(--anchor-padding));
+      z-index: 1;
+    }
+  }
```

Comment:
Feels a bit odd to have one bit of styling here, and the fallback in another file. Is that avoidable?

Reply:
resolved